public/scripts/project.js

```diff
@@ -1,94 +1,99 @@
 'use strict';
```

Comment:
I love how much you've commented out your code, it makes it very easy to tell exactly what each function does.